docker/Dockerfile

```diff
@@ -25,13 +28,13 @@ RUN apt-get update && apt-get install -y \
     && rosdep install --from-paths src --ignore-src -r -y \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install flask flask_restplus gevent rospkg catkin_pkg
+RUN sudo -H pip3 install flask flask_restplus gevent rospkg
```

Comment:
why do you need sudo?

Reply:
to avoid some warnings, it doesn't hurt anything, just make it look nicer when compile